providers/azure.go

```diff
@@ -90,15 +90,16 @@ func NewAzureProvider(p *ProviderData, opts options.AzureOptions) *AzureProvider
 	isV2Endpoint := false
 	if strings.Contains(p.LoginURL.String(), "v2.0") {
 		isV2Endpoint = true
+		azureV2GraphScope := fmt.Sprintf("https://%s/.default", p.ProfileURL.Host)
```

Comment:
Just to make sure. This is going to work for china and the rest as well because we set the default `graph.microsoft.com` url host as part of the `ProviderDefaults` in `NewAzure`